qualtran/bloqs/util_bloqs_test.py

```diff
@@ -55,6 +57,32 @@ def test_util_bloqs():
     assert no_return is None
 
 
+@frozen
+class TestPartition(Bloq):
+    test_bloq: Bloq
+
+    @cached_property
+    def bitsize(self):
+        return sum(reg.total_bits() for reg in self.test_bloq.signature)
+
+    @cached_property
+    def signature(self) -> Signature:
+        return Signature.build(regs=self.bitsize)
```

Comment:
can we use a different name than `regs`? i.e. there's a distinction between the register name and the Partition class attribute that we should clearly delineate in the test